taskcluster/.shared.yml

```diff
@@ -1,18 +1,28 @@
 python:
-  packages:
+  packages_trusty:
     apt: 'make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev'
+  packages_stretch:
+    apt: 'make build-essential libssl1.0-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev'
```

Comment:
We need libssl1.0 for Python build, and on stretch the libssl-dev package is not 1.0 :)